bundle/Controller/UserController.php

```diff
@@ -181,11 +182,15 @@ public function editAction(
             }
         }
 
+        $location = $this->locationService->loadLocation($user->versionInfo->contentInfo->mainLocationId);
```

Comment:
User could be assigned to multiple locations (user group) and it is possible situation when main location is not accessible here (by permissions). This sound like an edge case but we have 3. line issue https://jira.ez.no/browse/EZP-31310

Reply:
Corrected